### ISSUE TITLE
Cannot hide sliding panel when ListView is empty 

### ISSUE THREAD

**Issue by SeekMyCosmic** [umano/AndroidSlidingUpPanel#747]
Sometimes there is no data to initialize the listview, so I need to hide the panel, but it is impossible.
Even if the listview has one item, the sliding panel can be hide.
Please help me, how to hide the panel when the listview has nothing.

Maybe this is a BUG, hope the author to repair it. 

Thanks!

**Comment by tokudu** [comment]
setPanelState to HIdden when the listview is empty

**Comment by SeekMyCosmic** [comment]
Thank you for your reply!
About the version 3.3.0, I have tried many times to setPanelState to Hidden, but it's not impossible. However when I filled one piece of data in the list, the sliding panel can be hidden correctly.  You see that's not good enough and it must show it always before data is available.
I think the best way probably to do is that I can hide the panel first before the data is available and show it after the data prepared, or keep hidden state when the data is empty here.
I'll try the new version 3.3.1 and feedback here again.
Thanks for your help and you hard work that make things better and better!
